src/webui/static/app/dvr.js

```diff
@@ -56,7 +56,7 @@ tvheadend.dvrDetails = function(uuid) {
         if (autorec_caption)
           content += '<div class="x-epg-meta"><div class="x-epg-prefix">' + _('Autorec') + ':</div> ' + autorec_caption + '</div>';
         if (timerec_caption)
-          content += '<div class="x-epg-meta"><div class="x-epg-prefix">' + _('Time Scheduler') + ':</div> ' + timerec_caption + '</div>';
+          content += '<div class="x-epg-meta"><div class="x-epg-prefix">' + _('Time scheduler') + ':</div> ' + timerec_caption + '</div>';
```

Comment:
This is "time scheduler" as in "Which time schedule rule triggered this", isn't it... I wonder if there's a better wording, e.g. "Autorec rule:" and "Time schedule rule:"